src/pineko/evolve.py

```diff
@@ -11,7 +12,7 @@
 import rich.box
 import rich.panel
 import yaml
-from eko.io.types import ScaleVariationsMethod
+from eko.io.types import EvolutionMethod, ScaleVariationsMethod
```

Comment:
It seems to me that you're not using even `EvolutionMethod` anywhere...

Reply:
[6ee6ddd](https://github.com/NNPDF/pineko/pull/75/commits/6ee6ddd3bfc28bd0ec3882c9970e4923a2502205)